Documentation/network/servicemesh/mTLS-auth/mTLS-auth.rst

```diff
@@ -0,0 +1,101 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_mTLS_auth:
+
+****
+mTLS
+****
+
+
+Mutual TLS, also known as mTLS or two-way TLS, is a secure communication protocol that ensures the authenticity, integrity, and confidentiality of data exchanged between two entities over a network. 
+Unlike traditional TLS, which involves a one-way authentication process where the server verifies the client's identity, mutual TLS adds an additional layer of security by requiring both the client and the server to authenticate each other.
```

Comment:
should the traditional TLS be  "which involves a one-way authentication process where the client verifies the server's identity"?